### ISSUE TITLE
Can't figure out how to actually install ha-dyson or ha-dyson-cloud

### ISSUE THREAD

**Issue by mhaeberli** [shenxn/ha-dyson#144]
The repo has been added to HACS.  But I can't figure out the next step

**Comment by wenjiavv** [comment]
same here:
This integration does not support configuration via the UI. If you followed this link from the Home Assistant website, make sure you run the latest version of Home Assistant.

Version | core-2022.4.7
-- | --
Installation Type | Home Assistant Container
Development | false
Supervisor | false
Docker | true
User | root
Virtual Environment | false
Python Version | 3.9.9
Operating System Family | Linux
Operating System Version | 4.4.180+
CPU Architecture | x86_64

**Comment by amirook** [comment]
Same here:
This integration does not support configuration via the UI. If you followed this link from the Home Assistant website, make sure you run the latest version of Home Assistant.

Home Assistant Core 2022.6.5
Home Assistant Supervisor 2022.05.3
Home Assistant OS 8.2
Runung on RasPi

**Comment by Strax9** [comment]
I also have no idea what to do when the repo has been added to HACS. i run the latest version, any help would be greatly appreciated

**Comment by bdovaz** [comment]
Maybe related #138

**Comment by ghost** [comment]
Once you have added the repo in HACS, you then add it via the normal 'integrations' screen (the HACS repo makes it available like a normal integration).

edit: Clarification.  You need to add the integration per device in the normal 'integrations' screen.

**Comment by eholman89** [comment]
I'm having a similar issue, installed and downloaded Via HACS, and restarted my Pi but nothing shows up in the HA UI Integrations menu. But I must be doing something wrong as it's not just this Dyson integration. I'm having the same issue with a Tapo integration too.

**Comment by snigehere** [comment]
With hacs you usually have to clear browser cache before things show up in the integration page

**Comment by catalinbozianu** [comment]
Same here: "This integration does not support configuration via the UI. If you followed this link from the Home Assistant website, make sure you run the latest version of Home Assistant."

**Comment by mhaeberli** [comment]
Same problem.  Question - what are the next steps to configure?  Some stuff
in a text file?

which text file?

can someone please post a template?

Thanks!

**Comment by skrilla0** [comment]
Same problem here. Any resolution to install Dyson Cloud?

**Comment by vincemulhollon** [comment]
> Maybe related #138

I would agree with that assessment as I can reproduce amirook error message if I try to install a new integration while libdyson 0.8.7 is installed, although if I forcibly remove the libdyson 0.8.7 and install 0.8.11 and restart and then try to add a new dyson integration, then I get the normal working web pop up asking "setup using wifi or set up using manual" and all that as normal and if I owned a third fan I predict adding the integration would work.

I believe the "This integration does not support configuration via the UI." message and so forth is a red herring in the sense thats just a place holder message when the config crashes because libdyson 0.8.7 is installed and the integration requires 0.8.11 so it can't give you the web UI popups to configure the integration because its already crashed out.  Certainly I never configured my integrations using file, I used web UI.

I can 100% reliably reproduce both working and non-working situations mentioned above in this bug report by forcing the installation of libdyson 0.8.7 or 0.8.11 at HA startup time.  0.8.7 results in problems seen in this bug report and 0.8.11 works perfectly.

I am running HA in a docker container on Intel Ubuntu 20.04 server so I am changing/forcing the libdyson version inside my docker container not the dockerhost.  You all, as a group, seem to be using a variety of other HA installation methods, I think.

#138 has something else going on where something late in the start up process of HA seems to overwrite my fresh install of 0.8.11 with an older 0.8.7 which makes the SECOND restart after a forced installation of 0.8.7 fail.  So that aspect of the problem is weird, but probably not related.

**Comment by hscissors** [comment]
@vincemulhollon I tried force updating `libdyson` to 0.8.11 via `pip`. After restarting I saw Dyson Local, but it still errors:
```
 Logger: homeassistant.config_entries
Source: config_entries.py:784
First occurred: 5:27:14 PM (2 occurrences)
Last logged: 5:27:22 PM

    Error occurred loading configuration flow for integration dyson_cloud: cannot import name 'DysonPurifierHumidifyCoolFormaldehyde' from 'libdyson' (/usr/local/lib/python3.10/site-packages/libdyson/__init__.py)
    Error occurred loading configuration flow for integration dyson_local: cannot import name 'DysonPurifierHumidifyCoolFormaldehyde' from 'libdyson' (/usr/local/lib/python3.10/site-packages/libdyson/__init__.py)

```

**Comment by gmcaesar** [comment]
i just commented out line 15 in the init file and it seems to work for me, but hope you don't need that model......
